web/admin.py

```
@@ -157,10 +166,10 @@ class CustomUserAdmin(BaseUserAdmin):
         "formatted_date_joined",
         "formatted_last_login",
     )
-    list_filter = BaseUserAdmin.list_filter + (EmailVerifiedFilter, "date_joined", "last_login")
+    list_filter: ClassVar[tuple] = BaseUserAdmin.list_filter + (EmailVerifiedFilter, "date_joined", "last_login")
```

Ruff (0.8.2)

Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)